File: onresume.py
import frida, sys

def on_message(message, data):
    if message['type'] == 'send':
        print(message['payload'])
    else:
        print(message)

jscode = """
setImmediate(function(){
	Java.perform(function(){
		var Activity=Java.use("android.app.Activity")
		Activity.onResume.implementation = function(){
			send("[*] onResume() got called!");
			this.onResume();
		};
	});
});
"""
session = frida.get_usb_device().attach("package.name")
script = session.create_script(jscode)
script.on('message', on_message)
script.load()
sys.stdin.read()

File: frida_script/onresume.js
Java.perform(function(){
	var Activity = Java.use("android.app.Activity")
	Activity.onResume.implementation = function(){
		console.log("[*] onResume() got called!"); //console log 출력
		this.onResume(); //기존 함수 로직 실행
	}
})